### ISSUE TITLE
show,DataFrame throws error on DataFrame with nrow > 1

### ISSUE THREAD

**Issue by jorainer** [Bioconductor/S4Vectors#109]
Dear developers,

With the recent version of `S4Vectors` (0.35.3) I get an error with the `show` methods on `DataFrame`s with `nrow` > 1:

```r
> library(S4Vectors)
> DF <- DataFrame(a = 1:4, b = letters[1:4])
> show(DF)
DataFrame with 4 rows and 2 columns
Error in h(simpleError(msg, call)) : 
  error in evaluating the argument 'x' in selecting a method for function 'as.matrix': dims [product 8] do not match the length of object [2]
```
My session information (I use the *bioconductor/bioconductor_docker:devel*):

```r
> sessionInfo()
R version 4.2.1 (2022-06-23)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.5 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/openblas-pthread/libblas.so.3
LAPACK: /usr/lib/x86_64-linux-gnu/openblas-pthread/liblapack.so.3

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              
 [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
[1] Spectra_1.7.2        ProtGenerics_1.29.0  BiocParallel_1.31.12
[4] S4Vectors_0.35.3     BiocGenerics_0.43.2  BiocStyle_2.25.0    
[7] rmarkdown_2.16      

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.9          bslib_0.4.0         compiler_4.2.1     
 [4] BiocManager_1.30.18 jquerylib_0.1.4     tools_4.2.1        
 [7] ncdf4_1.19          digest_0.6.29       jsonlite_1.8.0     
[10] evaluate_0.16       clue_0.3-61         rlang_1.0.5        
[13] cli_3.3.0           yaml_2.3.5          parallel_4.2.1     
[16] xfun_0.32           fastmap_1.1.0       stringr_1.4.1      
[19] knitr_1.40          cluster_2.1.4       sass_0.4.2         
[22] fs_1.5.2            IRanges_2.31.2      MsCoreUtils_1.9.1  
[25] Biobase_2.57.1      R6_2.5.1            mzR_2.31.1         
[28] magrittr_2.0.3      codetools_0.2-18    htmltools_0.5.3    
[31] MASS_7.3-58.1       stringi_1.7.8       cachem_1.0.6       
```

**Comment by hpages** [comment]
This was fixed yesterday in **BiocGenerics** 0.43.3. This affected a lot of packages on yesterday's build report but it should go away today.

**Comment by jorainer** [comment]
Great! Thanks Herve! So I'll close
